co-log/src/Colog/Actions.hs

```diff
@@ -62,8 +67,13 @@ withLogTextFile path action = withFile path AppendMode $ action . logTextHandle
 -- Logger rotation
 ----------------------------------------------------------------------------
 
-{-
-data Limit = LimitTo Natural | Unlimited
+
+data Limit = LimitTo Natural | Unlimited deriving (Eq)
```

Comment:
Actually, if you write constructors in this particular order, the derived `Ord` instances will be correct, no need to write it manually